FrontEnd/src/components/adminPage/checkIfStaff.js

```diff
@@ -0,0 +1,15 @@
+import axios from 'axios';
+
+const checkIfStaff = async () => {
```

Comment:
Is seems that this function checks whether the user is an admin. Should it be reflected in the name of the function?

Reply:
rebuild => rabyld async function checkIfStaff() {